.azure-pipelines/common-templates/checkout-sdk.yml

```diff
@@ -0,0 +1,10 @@
+steps:  
```

Comment:
Why do we need to create a new checkout template? We already have one in https://github.com/microsoftgraph/msgraph-sdk-powershell/blob/dev/.azure-pipelines/common-templates/checkout.yml. Let's reuse it instead.